libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -927,6 +952,26 @@ instance ToJSON TeamDeleteData where
       [ "password" .= _tdAuthPassword tdd
       ]
 
+instance ToJSON LimitedTeamSize where
+  toJSON limitedSize =
+    let sizePairs =
+          case _ltsSize limitedSize of
+            Nothing ->  [ "within-limit" .= False ]
+            Just n ->  [ "within-limit" .= True, "size" .= n ]
+    in object (["limit" .= _ltsLimit limitedSize] ++ sizePairs )
```

Comment:
This would be nicer if we had a combinator that removes all `null` values from an object.  I wonder if that exists?  Anyway not important.

Reply:
Yeah, I was thinking about it, then I decided it is fine.